### PR TITLE
[TEST](bangc-ops): add mutual_information_backward api test

### PR DESCRIPTION
**Pull request by wangrt1**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

Please briefly describe what modification is made in this pull request, and indicate where to make the modification.

### CODE REVIEW

**Review by GuangpingYuan — APPROVED** [review]
LGTM

**Review by GuangpingYuan — APPROVED** [review]
LGTM

File: bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/mutual_information_backward/mutual_information_backward_workspace.cpp [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    setParam(true, true, false, true, true, true, true);
+    EXPECT_EQ(compute(), MLUOP_STATUS_BAD_PARAM);
+  } catch (std::exception &e) {
+    FAIL() << "MLUOPAPIGTEST: catched " << e.what()
+           << " in mutual_information_backward_workspace";
+  }
+}
+
+TEST_F(mutual_information_backward_workspace,
+       BAD_PARAM_opt_boundary_desc_null) {
```
Comment (tcd-fang) [inline bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/mutual_information_backward/mutual_information_backward_workspace.cpp:179]: opt_boundary_desc可以为空，无需添加

Comment (wangrt1) [reply bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/mutual_information_backward/mutual_information_backward_workspace.cpp]: 已去除

**Review by pingjiang123 — APPROVED** [review]
LGTM

**Pull request merged**